airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/OracleIntegrationTest.java

```diff
@@ -0,0 +1,222 @@
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.oracle;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.Database;
+import io.airbyte.db.Databases;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import io.airbyte.integrations.standardtest.destination.LocalAirbyteDestination;
+import io.airbyte.workers.protocols.airbyte.AirbyteDestination;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.jooq.JSONFormat;
+import org.jooq.JSONFormat.RecordFormat;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.OracleContainer;
+
+public class OracleIntegrationTest extends DestinationAcceptanceTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(OracleIntegrationTest.class);
+  private static final JSONFormat JSON_FORMAT = new JSONFormat().recordFormat(RecordFormat.OBJECT);
+
+  private static OracleContainer db;
+  private ExtendedNameTransformer namingResolver = new OracleNameTransformer();
+  private JsonNode config;
+
+  @BeforeAll
+  protected static void init() {
+    db = new OracleContainer("epiclabs/docker-oracle-xe-11g");
+    db.start();
+  }
+
+  @Override
+  protected String getImageName() {
+    return "airbyte/destination-oracle:dev";
+  }
+
+  @Override
+  protected AirbyteDestination getDestination() {
+    return new LocalAirbyteDestination(new OracleDestination());
+  }
+
+    private JsonNode getConfig(OracleContainer db) {
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("host", db.getHost())
+        .put("port", db.getFirstMappedPort())
+        .put("username", db.getUsername())
+        .put("password", db.getPassword())
+        .put("schema", "testSchema")
```

Comment:
there's a schema in these tests but not in the spec.json? something doesn't seem right here. based on the docs I think you wanted to go with supporting name spaces (hence the create databases privileges), which I'm supportive of, but then it needs to be in the spec.

Reply:
You're right.  I thought I left it in! 🤦‍♂️

Reply:
it's a bit scary that tests were all still passing despite this being the case. it would be helpful to identify how we could have caught this programmatically.

Reply:
I think it's as simple as "there's nothing in the system that validates this generated config object in the test system against the destination spec."  Should there be?

Reply:
yeah. i think there should be. would be worth creating creating an issue for this, i think.